src/options/options.vue

```diff
@@ -1,106 +1,161 @@
+              <input
+                class="input"
+                type="text"
+                maxlength="25"
+                placeholder="Title of the dashboard"
+                v-model="options.dashboards[0].title"
+              />
+            </div>
+          </div>
+        </div>
 
-          <div class="columns">
-            <div class="column preview">
-              <div class="field">
-                <label class="label">Height</label>
-                <div class="control">
-                  <input v-model="options.height" type="range" min="25" max="655" value="300" />
-                </div>
-              </div>
+        <div
+          class="columns"
+          v-for="(dashboard, index) in options.dashboards"
+          v-if="options.dashboards[0] !== dashboard"
+        >
+          <div class="field column" style="flex: 2">
+            <label class="label">Additional URL</label>
+            <div class="control">
+              <input
+                class="input"
+                type="text"
+                placeholder="URL of the additional Home Assistant page"
+                v-model="dashboard.url"
+              />
             </div>
-            <div class="column">
-              <div class="field">
-                <label class="label">Width</label>
-                <div class="control">
-                  <input v-model="options.width" type="range" min="25" max="800" value="300" />
-                </div>
-              </div>
+          </div>
+          <div class="field column" style="flex: 1">
+            <label class="label">Additional Title</label>
+            <div class="control">
+              <input
+                class="input"
+                type="text"
+                maxlength="25"
+                placeholder="Title of the dashboard"
+                v-model="dashboard.title"
+              />
             </div>
           </div>
+          <div class="column" style="flex: none">
+            <button class="button is-delete" @click="removeDashboard(index)">Remove</button>
+          </div>
+        </div>
 
-          <div class="columns">
-            <div class="column">
-              <div class="field">
-                <label class="label">Header visibility</label>
-                <label class="checkbox p-2">
-                  <input type="checkbox" v-model="options.hideHeader" />
-                  Hide header
-                </label>
+        <div>
+          <button
+            class="button is-primary addDashboardButton"
+            @click="addDashboard"
+            :disabled="dashboardsLimitReached()"
+          >
+            Add dashboard
+          </button>
+        </div>
+
+        <div class="columns">
+          <div class="column">
+            <div class="field">
+              <label class="label">Height</label>
+              <div class="control">
+                <input v-model="options.height" type="range" min="25" max="600" value="300" />
```

Comment:
I changed max value from 655 to 600, source: https://source.chromium.org/chromium/chromium/src/+/master:chrome/browser/ui/views/extensions/extension_popup.cc;drc=084514a32ae45d693e7068b4b63bafcc0aa85b63;l=36